samples/quickstart.js

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 'use strict';
+/*eslint no-warning-comments: [0, { "terms": ["todo", "fixme"], "location": "anywhere" }]*/
```

Comment:
do we really need this? (here and in other files)

Reply:
`npm run lint` complained if deleted this line
```
warning  Unexpected 'todo' comment  no-warning-comments
```